8_class_components/src/style.css

```diff
@@ -0,0 +1,243 @@
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+}
+
+.nav-links {
+	display: flex;
+	flex-direction: row;
+	gap: 20px;
+}
+
+.link {
+	color: #fff;
+}
+
+.logo {
+	display: flex;
+	align-items: center;
+}
+
+.logo h1 {
+	margin-left: 5px;
+}
+
+.logo img {
+	width: 40px;
+	height: 40px;
+}
+
+.search-container {
+	margin-top: 40px;
+	margin-bottom: 10px;
+	display: flex;
+	justify-content: center;
+}
+
+.search-bar {
+	border: 1px solid rgba(149, 157, 165, 0.2);
+	border-radius: 5px;
+	outline: 0;
+	padding: 10px 20px;
+	font-size: 14px;
+	height: 40px;
+	line-height: 24px;
+	width: 350px;
+}
+
+/* content - card */
+
+.card-container {
+	display: flex;
+	flex-wrap: wrap;
+	padding: 20px;
+	justify-content: center;
+}
+
+.card-container > a {
+	display: contents;
+}
+
+.card {
+	width: 350px;
+	max-width: 100%;
+	margin: 20px;
+	border-radius: 15px;
+	overflow: hidden;
+	box-shadow: rgba(149, 157, 165, 0.2) 0px 8px 24px;
+	border: none;
+	padding-top: 30px;
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	align-items: center;
+	text-align: center;
+}
+
+.card-body {
+	padding: 1.25rem;
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+}
+
+.card-footer {
+	background-color: #eff1f1;
+	width: 100%;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	gap: 10px;
+	padding: 20px;
+}
+
+.card-title {
+	margin-top: 7px;
+	margin-bottom: 5px;
+	font-weight: 700;
+	color: #192734;
+}
+
+.member-img {
+	height: 170px;
+	max-width: 100%;
+	object-fit: cover;
+	border: 1px solid #e1e3e5;
+	border-radius: 50%;
+	padding: 7px;
+}
+
+.followers {
+	margin-top: 7px;
+	font-size: 15px;
+}
+
+.text-muted {
+	color: #74808b;
+}
+
+.details {
+	display: flex;
+	flex-direction: column;
+	gap: 10px;
+	font-size: 15px;
+	color: #192734;
+	margin-top: 20px;
+	margin-bottom: 20px;
+}
+
+.detail-row {
+	display: flex;
+	flex-direction: row;
+	justify-content: baseline;
+	align-items: flex-end;
+	gap: 5px;
+}
+
+.icon {
+	width: 18px;
+	height: 18px;
+}
+
+/* No results */
+
+.centered-div {
+	display: flex;
+	flex-direction: column;
+	gap: 30px;
+	align-items: center;
+	justify-content: center;
+	height: 80vh;
+	padding: 15px;
+	line-height: 30px;
+}
+
+/* Member Details page */
+
+.member-info {
+	display: flex;
+	flex-direction: column;
+	padding: 100px;
+	justify-content: center;
+	align-items: left;
+}
+
+.profile {
+	display: flex;
+	gap: 30px;
+	align-items: center;
+	padding-bottom: 50px;
+	border-bottom: 1px solid #e1e3e5;
+}
+
+.profile .bio {
+	line-height: 30px;
+}
+
+.profile .member-img {
+	height: 250px;
+	max-width: 100%;
+}
+
+.profile .details {
+	margin-top: 10px;
+}
+
+.member-details {
+	display: flex;
+	flex-direction: column;
+	gap: 15px;
+	justify-content: space-between;
+}
+
+.stats {
+	padding-top: 50px;
+	display: flex;
+	flex-direction: row;
+	justify-content: space-evenly;
+}
+
+/* About us */
+
+.emoji {
+	font-size: 50px;
+}
+
+button {
```

Comment:
Add this html tag selector to the top having global style and then add custom styles using classes according to different scenarios of the respective button.